android/src/main/kotlin/com/example/flutter_video_compress/Utility.kt

```diff
@@ -0,0 +1,112 @@
+package com.example.flutter_video_compress
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.media.MediaMetadataRetriever
+import android.net.Uri
+import io.flutter.plugin.common.MethodChannel
+import org.json.JSONObject
+import java.io.File
+
+class Utility(private val channelName: String) {
+
+    fun deleteExists(file: File) {
```

Comment:
no need for Exists in name. Can file be null?

Reply:
I agree. I will rename to deleteFile.
No, file shouldn't be null. What will be the point of that method if it's null?